src/test/java/org/broadinstitute/hellbender/engine/CRAMReferenceIntegrationTest.java

```diff
@@ -1,82 +0,0 @@
-package org.broadinstitute.hellbender.engine;
-
-import org.broadinstitute.hellbender.CommandLineProgramTest;
-import org.broadinstitute.hellbender.exceptions.UserException;
-import org.broadinstitute.hellbender.tools.PrintReads;
-import org.broadinstitute.hellbender.utils.read.SamAssertionUtils;
-import org.broadinstitute.hellbender.utils.test.BaseTest;
-import org.testng.Assert;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
-
-import java.io.File;
-
-public final class CRAMReferenceIntegrationTest extends CommandLineProgramTest{
```

Comment:
Note that the tests in the class weren't deleted -- the class was just renamed to `CRAMSupportIntegrationTest`, but github is failing to detect the rename.